cmd/webhook/builder.go

```diff
@@ -2,12 +2,12 @@ package webhook
 
 import (
 	"context"
+	dynatracev1beta2 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
```

Comment:
linter?

Reply:
pls read the description first 😬 

> ⚠️This is not a complete, merge and you are done change, the linting and tests have been messed up, and there are still changes that need to be finished.

as this is not my ticket I didn't want to do everything, this is not aimed at `main` but another PR.